app/models/mixins/inter_region_api_method_relay.rb

```diff
@@ -70,8 +70,11 @@ def self.api_client_connection_for_region(region_number, user = User.current_use
 
   def self.exec_api_call(region, collection_name, action, api_args = nil, id = nil)
     require 'manageiq-api-client'
+    _log.info("Executing InterRegion api call for region #{region} with arguments: #{api_args}")
```

Comment:
Could these args have something sensitive in them that we maybe don't want to log? I think the api logs have a special parser that sanitizes things which we wouldn't have here.

Reply:
Ohh, ok. I wanted to provide more info for future troubleshooting.
Will remove logging